package.json

```
@@ -15,5 +15,8 @@
   "devDependencies": {
     "husky": "^8.0.3",
     "turbo": "^1.10.12"
+  },
+  "dependencies": {
+    "next-intl": "^2.20.2"
```

```suggestion
  }
```
the dependency should not be installed here, but you should add it into its app `nextjs-website`